hypervisor/arch/x86/cpu.c

```diff
@@ -558,9 +561,6 @@ static void bsp_boot_post(void)
 	/* Start all secondary cores */
 	start_cpus();
 
-	/* Trigger event to allow secondary CPUs to continue */
-	bitmap_set_nolock(0U, &pcpu_sync);
-
```

Comment:
why we clear the reset part?

Reply:
It's done by atomic_store64(&pcpu_sync, PCPU_SYNC_RUN); in start_cpus.